app/models/person_escort_record.rb

```diff
@@ -51,6 +51,7 @@ def self.save_with_responses!(profile_id: nil, version: nil, move_id: nil)
 
     record = new(profile: profile, move: move, framework: framework)
     record.build_responses!
+    record.import_nomis_mappings!
```

Comment:
might be more readable as
```
record.import_nomis_mappings! if move&.from_location&.prison?
record
...
```

Reply:
good suggestion I tidied it up slightly,  however l left the check in the method itself as wanted to encapsulate the import logic into the `import_nomis_mappings` method

Reply:
👍